mocks/flask-react-mocks/server/sampleData.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+
+shortLoremIpsum = "Lorem id sint aliqua tempor tempor sit. Ad dolor dolor ut nulla mollit dolore non eiusmod Lorem tempor nisi cillum."
+longLoremIpsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Risus at ultrices mi tempus imperdiet."
+
+sampleGridData = [
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': 'Company A',
+    'status': 'Closed',
+    'shipTo': 'Francisco Pérez-Olaeta',
+    'orderTotal': 2490.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 1
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': 'Company B',
+    'status': 'Closed',
+    'shipTo': 'Soo Jung Lee',
+    'orderTotal': 1760.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 2
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': 'Company C',
+    'status': 'Shipped',
+    'shipTo': 'Run Liu',
+    'orderTotal': 665.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 3
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company D",
+    'status': "Shipped",
+    'shipTo': "Soo Jung Lee",
+    'orderTotal': 560.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 4
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company E",
+    'status': "New",
+    'shipTo': "John Rodman",
+    'orderTotal': 810.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 5
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company F",
+    'status': "New",
+    'shipTo': "Elizabeth Andersen",
+    'orderTotal': 196.5,
+    'orderDate': datetime.datetime.now(),
+    'id': 6
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company G",
+    'status': "Closed",
+    'shipTo': "Peter Krschne",
+    'orderTotal': 270.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 7
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company H",
+    'status': "Closed",
+    'shipTo': "Sven Mortensen",
+    'orderTotal': 736.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 8
+  },
+  {
+    'shortDescription': shortLoremIpsum,
+    'longDescription': longLoremIpsum,
+    'title': "Company I",
+    'status': "Shipped",
+    'shipTo': "Anna Bedecs",
+    'orderTotal': 800.0,
+    'orderDate': datetime.datetime.now(),
+    'id': 9
+  }
+]
+
+sampleData = {
+    'listTextAssets': [
```

Comment:
this diverges a bit from the node data:
```
sampleData.listID = 3;

sampleData.listTextAssets = [
  {
    text: shortLoremIpsum,
    _id: 1
  },
  {
    text: shortLoremIpsum,
    _id: 2
  }
];
```
Was this intentional?

Reply:
No, this was not intentional. I just put some random text to keep it simple but forgot to update it later. Shall update now.

Reply:
Cool!